woof-code/rootfs-skeleton/usr/sbin/sfs_load

```diff
@@ -49,7 +49,13 @@
 
 . /etc/rc.d/PUPSTATE
 
-if [ -L /bin -o -L /sbin -o -L /lib -o -L /lib64 -o -L /lib32 -o -L /libx32 ]; then
+sym_list=""
+for fld2 in bin sbin lib lib64 lib32 libx32
+do
+  [ -L /$fld2 ] && sym_list="$sym_list $fld2"
+done
+
+if [ "$sym_list" != "" ]; then
```

Comment:
This entire block should be after `if [ "$1" ]; then`: there's no need to slow down sfs_load if we're not loading a SFS